nidx/nidx_protos/src/nidx.proto

```diff
@@ -33,7 +33,6 @@ service NidxSearcher {
   rpc DocumentSearch(nodereader.DocumentSearchRequest) returns (nodereader.DocumentSearchResponse) {}
   rpc ParagraphSearch(nodereader.ParagraphSearchRequest) returns (nodereader.ParagraphSearchResponse) {}
   rpc VectorSearch(nodereader.VectorSearchRequest) returns (nodereader.VectorSearchResponse) {}
-  rpc RelationSearch(nodereader.RelationSearchRequest) returns (nodereader.RelationSearchResponse) {}
```

Comment:
Maybe we can remove DocumentSearch, ParagraphSearch, etc. as well?

Reply:
Yes, I was working on it :)

Reply:
Done